sources/Makefile.am

```diff
@@ -13,9 +13,9 @@ sources =
 
 build_sources = AssemblyInfo.cs $(sources)
 
-CLEANFILES = $(DLL) generated-stamp generated/*.cs $(API)
+CLEANFILES = $(DLL) generated-stamp generated/* $(API)
 
-DISTCLEANFILES = AssemblyInfo.cs $(DLLMAP)
+DISTCLEANFILES = AssemblyInfo.cs
```

Comment:
This is a "sister" change from the EXTRA_DIST change above. The original commit message isn't very obvious (just says don't clear the dll map). Maybe some specific distro change where the dll map is custom managed or something?